models/Program.js

```diff
@@ -1,5 +1,5 @@
-const mongoose = require("mongoose");
-const Schema = mongoose.Schema;
```

Comment:
why

Reply:
didn't delete that. Not sure why it is showing that

Reply:
yeah idk where this even is anymore